README.md

```diff
@@ -204,6 +204,95 @@ add_filter( 'packeta_order_grid_links_settings', function ( \Packetery\Module\Or
+| `[packeta_tracking_url order_id="<?php echo $order->get_id(); ?>"]`         | Displays the tracking URL if available              |
+| `[packeta_pickup_point_id order_id="<?php echo $order->get_id(); ?>"]`      | Displays the pickup point ID if available           |
+| `[packeta_pickup_point_name order_id="<?php echo $order->get_id(); ?>"]`    | Displays the pickup point name if available         |
+| `[packeta_pickup_point_address order_id="<?php echo $order->get_id(); ?>"]` | Displays the full pickup point address if available |
+| `[packeta_pickup_point_street order_id="<?php echo $order->get_id(); ?>"]`  | Displays the pickup point street if available       |
+| `[packeta_pickup_point_city order_id="<?php echo $order->get_id(); ?>"]`    | Displays the pickup point city if available         |
+| `[packeta_pickup_point_zip order_id="<?php echo $order->get_id(); ?>"]`     | Displays the pickup point ZIP if available          |
+| `[packeta_pickup_point_country order_id="<?php echo $order->get_id(); ?>"]` | Displays the pickup point country if available      |
+| `[packeta_carrier_name order_id="<?php echo $order->get_id(); ?>"]`         | Displays the carrier name if available              |
+
+### Conditional Shortcodes
+
+You can use conditional shortcodes to display content only when certain conditions are met:
+
+#### If Submitted
+
+Use this to display content only when the order has been submitted to Packeta:
+
+```php
+[packeta_if_packet_submitted order_id="<?php echo $order->get_id(); ?>"]
+	Your tracking number is: [packeta_tracking_number order_id="<?php echo $order->get_id(); ?>"]
+	Track your package here: [packeta_tracking_url order_id="<?php echo $order->get_id(); ?>"]
+[/packeta_if_packet_submitted]
+```
+
+#### If Pickup Point
+
+Use this to display content only when the order uses a pickup point:
+
+```php
+[packeta_if_pickup_point order_id="<?php echo $order->get_id(); ?>"]
+	Your pickup point is: [packeta_pickup_point_name order_id="<?php echo $order->get_id(); ?>"]
+	Address: [packeta_pickup_point_address order_id="<?php echo $order->get_id(); ?>"]
+[/packeta_if_pickup_point]
+```
+
+#### If carrier
+
+Use this to display content only when the order has a carrier assigned:
+
+```php
+[packeta_if_carrier order_id="<?php echo $order->get_id(); ?>"]
+	Your order is shipped with: [packeta_carrier_name order_id="<?php echo $order->get_id(); ?>"]
+[/packeta_if_carrier]
+```
+
+### Usage in Email Templates
+
+You can use these shortcodes in your WooCommerce email templates. To add them to your email templates:
+
+1. Go to WooCommerce > Settings > Emails
+2. Click on any email template you want to customize (e.g., "New Order", "Processing Order", etc.)
+3. In the "Email content" section, you can add the shortcodes to customize the email content
+
+For example:
+
+```php
+<h2>Order Details</h2>
+<p>Order #<?php echo $order->get_order_number(); ?></p>
```

Comment:
Ten komentář je záměrný?

Reply:
Takhže to není php, ale html.

Reply:
Aby se to správně zvýrazňovalo, musí být na začátku bloku `<?php` - html i neuvedení jazyka se mi zdá lepší, než aktuální stav.

Reply:
Github to vidí stejně. Je mi celkem jedno, které řešení vedoucí k tomu, aby se nezobrazovalo rozbitě, bude použito. Případně probereme s teamem.